frontend/src/components/Buttons/ScrollToTopButton.vue

```diff
@@ -2,11 +2,11 @@
   <v-fab-transition>
```

Comment:
This button should be rounded, like the play one (if we want to strictly stick with master styles)